content/tutorial-regular-keys.md

```diff
@@ -0,0 +1,764 @@
+
+An example of the request format:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```
+{
+  "method": "wallet_propose"
+}
+```
+
+*JSON-RPC*
+
+```
+{
+  "method": "wallet_propose"
+}
+```
+
+*Commandline*
+
+```
+#Syntax: wallet_propose
+rippled wallet_propose
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+
+#### Response Format
+
+An example of a successful response:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```
+{
+  "result": {
+    "account_id": "rsprUqu6BHAffAeG4HpSdjBNvnA6gdnZV7",
+    "key_type": "secp256k1",
+    "master_key": "KNEW BENT LYNN LED GAD BEN KENT SHAM HOBO RINK WALT ALLY",
+    "master_seed": "sh8i92YRnEjJy3fpFkL8txQSCVo79",
+    "master_seed_hex": "966C0F68643EFBA50D58D191D4CA8AA7",
+    "public_key": "aBRNH5wUurfhZcoyR6nRwDSa95gMBkovBJ8V4cp1C1pM28H7EPL1",
+    "public_key_hex": "03AEEFE1E8ED4BBC009DE996AC03A8C6B5713B1554794056C66E5B8D1753C7DD0E"
+  },
+  "status": "success",
+  "type": "response"
+}
+```
+
+*JSON-RPC*
+
+```
+{
+    "result": {
+        "account_id": "rsprUqu6BHAffAeG4HpSdjBNvnA6gdnZV7",
+        "key_type": "secp256k1",
+        "master_key": "KNEW BENT LYNN LED GAD BEN KENT SHAM HOBO RINK WALT ALLY",
+        "master_seed": "sh8i92YRnEjJy3fpFkL8txQSCVo79",
+        "master_seed_hex": "966C0F68643EFBA50D58D191D4CA8AA7",
+        "public_key": "aBRNH5wUurfhZcoyR6nRwDSa95gMBkovBJ8V4cp1C1pM28H7EPL1",
+        "public_key_hex": "03AEEFE1E8ED4BBC009DE996AC03A8C6B5713B1554794056C66E5B8D1753C7DD0E"
+        "status": "success"
+    }
+}
+```
+
+*Commandline*
+
+```
+{
+   "result" : {
+      "account_id" : "rsprUqu6BHAffAeG4HpSdjBNvnA6gdnZV7",
+      "key_type" : "secp256k1",
+      "master_key" : "KNEW BENT LYNN LED GAD BEN KENT SHAM HOBO RINK WALT ALLY",
+      "master_seed" : "sh8i92YRnEjJy3fpFkL8txQSCVo79",
+      "master_seed_hex" : "966C0F68643EFBA50D58D191D4CA8AA7",
+      "public_key" : "aBRNH5wUurfhZcoyR6nRwDSa95gMBkovBJ8V4cp1C1pM28H7EPL1",
+      "public_key_hex" : "03AEEFE1E8ED4BBC009DE996AC03A8C6B5713B1554794056C66E5B8D1753C7DD0E",
+      "status" : "success"
+   }
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+In the next step, you'll use the `account_id` from this response to assign the key pair as a regular key pair to your account.
+
+
+### 2. Assign the Key Pair to Your Account as a Regular Key Pair
+
+Use the [`SetRegularKey`](reference-transaction-format.html#setregularkey) method to assign the key pair you generated in step 1 to your account as a regular key pair.
+
+When assigning a regular key pair to your account for the first time, the `SetRegularKey` method requires signing by your account's master private key (secret). Transmitting your master private key is dangerous, so we'll complete this transaction in two steps to keep transaction signing separate from transaction submission to the network.
+
+When you make subsequent `SetRegularKey` requests, you can sign using the existing regular private key to replace or [remove itself](#removing-or-changing-a-regular-key-pair).
```

Comment:
Maybe add a note to the effect that you should still not submit your regular key across the network, either.